shuup_tests/admin/test_views.py

```diff
@@ -293,6 +294,31 @@ def test_product_edit_view(rf, admin_user, settings):
     else:
         assert not shop_product.primary_category
 
+    # Test for showing alert of validation issues
+    view = ProductEditView.as_view()
+    request = apply_request_middleware(rf.get("/"), user=admin_user)
+    response = view(request, pk=shop_product.pk)
+    response.render()
+    content = force_text(response.content)
+    soup = BeautifulSoup(content, "lxml")
+    alert = soup.find_all("div", {"class": "validation-issues-alert"})
+    assert not alert
+
+    with override_provides(
+        "admin_product_validator",
+        ["shuup.testing.admin_product_validator:TestAdminProductValidator"],
+    ):
+        view = ProductEditView.as_view()
+        request = apply_request_middleware(rf.get("/"), user=admin_user)
+        response = view(request, pk=shop_product.pk)
+        response.render()
+        content = force_text(response.content)
+        soup = BeautifulSoup(content, "lxml")
+        alert = soup.find_all("div", {"class": "validation-issues-alert"})
```

Comment:
test that the warning issue that I commented above is rendering HTML as it should and the script tag is being removed